src/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -39,14 +35,19 @@ internal bool Sign
 
         internal int Scale
         {
-            get { return (int)((uflags & ScaleMask) >> ScaleShift); }
+            get { return (byte)(uflags >> ScaleShift); }
             set { uflags = (uflags & ~ScaleMask) | ((uint)value << ScaleShift); }
         }
 
         private ulong Low64
         {
+#if BIGENDIAN || DEBUG // On Debug builds, include the big-endian compatible code to help deter bitrot
```

Comment:
We prefer to use the same code for both debug and release builds

Reply:
I copied this from Enum.cs, but I'm happy to remove it.

Reply:
Ah, ok. You can fix the Enum.cs to be plain `#if BIGENDIAN` as well if you would like (it is what all other BIGENDIAN ifdefs look like).

Reply:
@pentp - The difference is that Enum.cs doesn't change the debug vs. release code path. The `Conditional` attribute prevents the BIG_ENDIAN code from running at all, but still forces C# to parse and error-check the code thus giving us half the loaf (compile-rot protection.) That's a pattern that was easy to use in that case - not such much here so making it `#if BIGENDIAN` only here is fine.